UNRELEASED.md

```diff
@@ -9,6 +9,8 @@
   - In Objective-C some options are no longer supported e.g. the ability to show a notification dot and the support for badges.
   - The component adopts the new `BPKIcon` API and enforces the correct usage of icons with the appropriate horizontal navigation size at compile time.
   - Due to a bug in Swift/UIKit with generic views the component no longer supports theming. Theming can still be achieved with an explicit wrapping view that integrates with `UIAppearance`.
+- Backpack/Chip:
+  - The type of `iconName` has changed from the deprecated `BPKIconName` to `BPKIconNameLarge`. Some icons previously used might to be available.
```

Comment:
This shouldn't be the case as using them would have resulted in a runtime error, but I guess it does no harm to mention

Reply:
Right, probably still worth mentioning?